LayoutTests/imported/w3c/web-platform-tests/svg/idlharness.window-expected.txt

```diff
@@ -1631,5 +1646,188 @@ PASS SVGElement interface: objects.view must inherit property "ownerSVGElement"
 PASS SVGElement interface: objects.view must inherit property "viewportElement" with the proper type
 FAIL SVGElement interface: objects.view must inherit property "correspondingElement" with the proper type assert_inherits: property "correspondingElement" not found in prototype chain
 FAIL SVGElement interface: objects.view must inherit property "correspondingUseElement" with the proper type assert_inherits: property "correspondingUseElement" not found in prototype chain
+FAIL TimeEvent interface: existence and properties of interface object assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface object length assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface object name assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: existence and properties of interface prototype object assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: existence and properties of interface prototype object's "constructor" property assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: existence and properties of interface prototype object's @@unscopables property assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: attribute view assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: attribute detail assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+FAIL TimeEvent interface: operation initTimeEvent(DOMString, Window?, long) assert_own_property: self does not have own property "TimeEvent" expected property "TimeEvent" missing
+PASS SVGAnimationElement interface: existence and properties of interface object
+PASS SVGAnimationElement interface object length
+PASS SVGAnimationElement interface object name
+PASS SVGAnimationElement interface: existence and properties of interface prototype object
+PASS SVGAnimationElement interface: existence and properties of interface prototype object's "constructor" property
+PASS SVGAnimationElement interface: existence and properties of interface prototype object's @@unscopables property
+PASS SVGAnimationElement interface: attribute targetElement
+PASS SVGAnimationElement interface: attribute onbegin
+FAIL SVGAnimationElement interface: attribute onend assert_true: The prototype object must have a property "onend" expected true got false
+FAIL SVGAnimationElement interface: attribute onrepeat assert_true: The prototype object must have a property "onrepeat" expected true got false
```

Comment:
Why do not you add 
```
attribute EventHandler onend;
attribute EventHandler onrepeat;
```
to the IDL to match the spec and to fix these failures?

Reply:
We need to do full `repeatEvent` implementation first - which is this PR - https://github.com/WebKit/WebKit/pull/29658 and for `onend`, it expose another bug and lead to WPT failure, which needs further investigation. So I am doing minimum, which is not regressing anything.